packages/components/atoms/f-form-field/stories/TextInputDefault.stories.js

```diff
@@ -1,77 +1,88 @@
-import {
-    withKnobs, select, text, boolean
-} from '@storybook/addon-knobs';
 import { withA11y } from '@storybook/addon-a11y';
 import FormField from '../src/components/FormField.vue';
+import SharedArgTypes from './SharedArgTypes';
 import {
+    DEFAULT_INPUT_TYPE,
     DEFAULT_FIELD_SIZE,
     VALID_FIELD_SIZES,
     VALID_TEXT_INPUT_TYPES
 } from '../src/constants';
 
 export default {
     title: 'Components/Atoms/f-form-field',
-    decorators: [withKnobs, withA11y]
+    decorators: [withA11y]
 };
 
-export const TextInputDefaultComponent = () => ({
+export const TextInputDefaultComponent = (args, { argTypes }) => ({
     components: { FormField },
-    props: {
-        locale: {
-            default: select('Locale', ['en-GB', 'en-AU'])
-        },
-        labelText: {
-            default: text('Label Text', 'First name')
-        },
-        fieldSize: {
-            default: select('Field Size', VALID_FIELD_SIZES, DEFAULT_FIELD_SIZE)
-        },
-        inputType: {
-            default: select('Input Type', VALID_TEXT_INPUT_TYPES)
-        },
-        isDisabled: {
-            default: select('isDisabled', [null, 'disabled'])
-        },
-        shouldShowLabelText: {
-            default: boolean('shouldShowLabelText', true)
-        },
-        hasError: {
-            default: boolean('hasError', false)
-        },
-        isGrouped: {
-            default: boolean('isGrouped', false)
-        },
-        labelDescription: {
-            default: text('Label Description', '')
-        },
-        labelDetails: {
-            default: text('Label Details', '')
-        },
-        assistiveText: {
-            default: text('Assistive Text', '')
-        },
-        placeholder: {
-            default: text('Placeholder', '')
-        }
-    },
-    parameters: {
-        notes: 'some documentation here'
-    },
+    props: Object.keys(argTypes),
     template:
         `<form-field
             :locale="locale"
             :label-text="labelText"
             :input-type="inputType"
-            :shouldShowLabelText="shouldShowLabelText"
+            :should-show-label-text="shouldShowLabelText"
             :field-size="fieldSize"
             :has-error="hasError"
             :is-grouped="isGrouped"
             :disabled="isDisabled"
             :placeholder="placeholder"
-            :maxlength="200"
-            :labelDetails="labelDetails"
-            :labelDescription="labelDescription"
-            :assistiveText="assistiveText"/>`
+            :max-length="200"
```

Comment:
I think this is the HTML attribute `maxlength` rather than a prop called `maxLength`

Reply:
To make it clearer, you can probably get rid of the binding too, and the same for `row` and `col` in the textarea component
```suggestion
            maxlength="200"
```